rdkit/Chem/SaltRemover.py

```diff
@@ -146,6 +196,13 @@ def StripMol(self, mol, dontRemoveEverything=False):
     2
 
     """
+    m, removed = self._StripMol(mol, dontRemoveEverything)
+    return m
+
+  def StripMolAndReturnRemoved(self, mol, dontRemoveEverything=False):
+    return self._StripMol(mol, dontRemoveEverything)
```

Comment:
You could actually use a `namedtuple` as the return value of _StripMol. Then the first one would be 
```
result = self._StripMol(mol, dontRemoveEverything)
return result.mol
```
and the other case
```
return self._StripMol(mol, dontRemoveEverything)
```